src/style.css

```
@@ -0,0 +1,22 @@
+@import url(./blocks/stars.css);
+@import url(./blocks/card.css);
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
+
+:root {
+  --card-background: #fff;
+  --card__title-color: #060b35;
+  --card__reviews-color: #060b35;
+  --card__product-price-value-color: #060b35;
+  --card__product-code-color: #616070;
+  --card__product-price-color: #616070;
+  --card__button-background-color: #00acdc;
```

Why do you use different variables for the same colors? BTW The background is white by default 
```suggestion
  --main-color: #060b35;
  --secondary-color: #616070;
  --button-color: #00acdc;
```